third_party/move/move-core/types/src/account_address.rs

```diff
@@ -18,6 +18,8 @@ impl AccountAddress {
     /// The number of bytes in an address.
     /// Default to 16 bytes, can be set to 20 bytes with address20 feature.
     pub const LENGTH: usize = 32;
+    /// Max address: 0xff....
+    pub const MAX_ADDRESS: Self = Self([0xFF; Self::LENGTH]);
```

Comment:
Does this notation actually fill up FF for all 32 bytes? If so, never stop learning about Rust

Reply:
Yes. Note that I used the same notation as was there before: I just changed the hard coded usages to instead use a public constant.